docs/content/querying/filters.md

```diff
@@ -174,26 +174,33 @@ The IN filter supports the use of extraction functions, see [Filtering with Extr
 
 ### Bound filter
 
-Bound filter can be used to filter by comparing dimension values to an upper value or/and a lower value. 
-By default Comparison is string based and **case sensitive**.
-To use numeric comparison you can set `alphaNumeric` to `true`.
-By default the bound filter is a not a strict inclusion `inputString <= upper && inputSting >= lower`.
+The Bound filter can be used to filter by comparing dimension values to an upper value and/or a lower value.
 
+|property|type|description|required?|
+|--------|-----------|---------|
+|type|String|This should always be "bound".|yes|
+|dimension|String|The dimension to filter on|yes|
+|lower|String|The lower bound for the filter|no|
+|upper|String|The upper bound for the filter|no|
+|lowerStrict|Boolean|Perform strict comparison on the lower bound ("<" instead of "<=")|no, default: false|
+|upperStrict|Boolean|Perform strict comparison on the upper bound (">" instead of ">=")|no, default: false|
+|ordering|String|Specifies the sorting order to use when comparing values against the bound.<br/>Can be one of the following values: "lexicographic", "alphanumeric", "numeric", "strlen".<br/>See [Sorting Orders](./sorting-orders.html) for more details.|no, default: "lexicographic"|
```

Comment:
Something is messing up the formatting here, maybe the embedded html (check out "View", the table doesn't render). Even if it's not the culprit, I don't think the `<br/>` are really needed, we could allow the browser's builtin word wrap do its thing.